src/sql/azdata.proposed.d.ts

```diff
@@ -1053,7 +1055,7 @@ declare module 'azdata' {
 		/**
 		 * String representation of graph
 		 */
-		graphFile: ExecutionPlanGraphFile;
+		graphFile: ExecutionPlanGraphInfo;
```

Comment:
Please consider have your own namespace, we have too many stuff in the root namespace. it is easier to do it now.